packages/config/src/__tests__/fixtures/language-support/js/app.json

```
@@ -0,0 +1,4 @@
+{
+  "foo": "invalid",
+  "slug": "someslug"
```

```suggestion
  "expo": {
    "slug": "someslug"
  }
```
(This is the format we support right now.)